test/server/cards/01_SOR/events/ForceThrow.spec.ts

```diff
@@ -0,0 +1,76 @@
+describe('Force Throw', function() {
+    integration(function(contextRef) {
+        describe('Force Throw\'s ability', function() {
+            it('should select opponent to discard a card and deal damage to a unit equals to its cost', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['force-throw'],
+                        groundArena: ['wampa', 'ezra-bridger#resourceful-troublemaker']
+                    },
+                    player2: {
+                        hand: ['karabast'],
+                        groundArena: ['specforce-soldier', 'atst'],
+                        spaceArena: ['tieln-fighter']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                // Damage dealt by opponent discarded card
+                context.player1.clickCard(context.forceThrow);
+                context.player1.clickPrompt('Opponent');
+                expect(context.karabast).toBeInZone('discard');
+                expect(context.player1).toBeAbleToSelectExactly([context.tielnFighter, context.specforceSoldier, context.atst, context.wampa, context.ezraBridger]);
+                context.player1.clickCard(context.atst);
+                expect(context.atst.damage).toBe(2);
+                expect(context.player2).toBeActivePlayer();
+            });
+
+            it('should select opponent to discard a card but no deal damage as there is not a force unit', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['force-throw'],
+                        groundArena: ['wampa']
+                    },
+                    player2: {
+                        hand: ['karabast'],
+                        groundArena: ['specforce-soldier', 'atst']
```

Comment:
We should add a Force unit on the opponent side to confirm that doesn't trigger the effect